### PR TITLE
Make photo creation time zone aware

### PR DESCRIPTION
**Pull request by philipbl**
Thanks for the great project! This is a PR regarding the creation time of photos.

Previously, the creation time of a photo would be in the local time zone and not time zone aware. This makes is hard to reason about the time the photo was created. This PR sets the time to UTC and makes it time zone aware.

Also switch to ISO 8601 as the output. This is what the pervious code was doing, except time zone information will be included.

Is there a reason why the the creation time is converted to a string? I think it would be more powerful to leave it as a `datetime` object. For this PR, I kept it as a string in case there was a reason for doing so.

### CODE REVIEW

File: pyicloud/services/photos.py [hunk 1]
```diff
@@ -213,8 +214,9 @@ def size(self):
 
     @property
     def created(self):
-        dt = datetime.fromtimestamp(self.data.get('createdDate') / 1000.0)
-        return dt.strftime('%Y-%m-%dT%H:%M:%SZ')
+        dt = datetime.utcfromtimestamp(self.data.get('createdDate') / 1000.0)
+        dt = dt.replace(tzinfo=pytz.utc)
```
Comment (torarnv) [inline pyicloud/services/photos.py]: Out of curiosity, why is this needed? `tzinfo=None` seems to indicate UTC by default?

Comment (philipbl) [inline pyicloud/services/photos.py]: You're right, this can be simplified to

```python
dt = datetime.fromtimestamp(self.data.get('createdDate') / 1000.0, tz=pytz.utc)
```

The time zone needs to specified somewhere or else the `datetime` object will not be time zone aware. I did some experimenting [here](https://gist.github.com/philipbl/dcd4b93f80fbc8f6f8e0dce25b3e49c4).

**Comment by philipbl** [comment]
I've updated the PR.

**Comment by torarnv** [comment]
Thanks!
